data/data_loader.py

```diff
@@ -0,0 +1,47 @@
+
+import os
+import sys
+import json
+
+import torch
+from torch.utils.data import Dataset, DataLoader
+
+top_config = {}
+cwd = os.getcwd()  # current working directory
+cfp = os.path.dirname(os.path.abspath(__file__))  # current file path
+os.chdir(cfp)
+root_full_path = os.path.abspath("..")
+sys.path.append(root_full_path)
+config_full_path = os.path.join(root_full_path, 'config/config.json')
+if not os.path.exists(config_full_path):
+    print(f"{config_full_path} doesn't exist.")
+    exit(0)
+with open(config_full_path, 'r') as file:
+    top_config = json.load(file)
+os.chdir(cwd)
+
+# TODO(diwei): Modify the class' detail including preproess, dataset and so on.
+class TopDataset(Dataset):
+    def __init__(self):
+        self.num = 10
+
+        self.tgts = torch.rand(top_config["model_params"]["tgt_length"],
```

Comment:
The dataset should be created for the source and target sequences, right? Mems represents the intermediate output of the encoder and should not be a part of the dataset. Pls correct me if I am wrong.